sqlmesh/core/console.py

```diff
@@ -469,6 +469,15 @@ def _show_options_after_categorization(self, plan: Plan, auto_apply: bool) -> No
             self._prompt_backfill(plan, auto_apply)
         elif plan.context_diff.has_changes and not auto_apply:
             self._prompt_promote(plan)
+        elif (
+            plan.is_dev
```

Comment:
i think we can dry this up, it can be a property on the plan that includes this logic except for auto_aply